suite-native/helpers/src/hooks/useRedirect.ts

```diff
@@ -0,0 +1,22 @@
+import { Linking } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { notificationsActions } from '@suite-common/toast-notifications';
+
+export const useRedirect = () => {
+    const dispatch = useDispatch();
+
+    const handleRedirect = async (href: string) => {
```

Comment:
please wrap this to `useCallback`, in this universal hook you never know how people will use it and you should provide stable callback function signature